### PR TITLE
Select styles

### PR DESCRIPTION
**Pull request by iAdramelk**

### CODE REVIEW

File: components/DocHeader/DocHeader.tsx [hunk 1]
```diff
@@ -48,7 +48,9 @@ const DocHeader = ({
           <Box as="h1" pr={6} color="white" fontWeight="regular">
             {title}
           </Box>
-          <DocHeaderVersions versions={versions} />
+          <Box mt={1}>
+            <DocHeaderVersions versions={versions} />
```
Comment (alex-kovoy) [inline components/DocHeader/DocHeader.tsx:52]: Lets rename DocHeaderVersions to:

```ts
import Versions, { VersionItem} from "./Versions"
```

This way it matches the file name and also shorter

File: components/DocHeader/DocHeader.tsx [hunk 2]
```diff
@@ -48,7 +48,9 @@ const DocHeader = ({
           <Box as="h1" pr={6} color="white" fontWeight="regular">
             {title}
           </Box>
-          <DocHeaderVersions versions={versions} />
+          <Box mt={1}>
```
Comment (alex-kovoy) [inline components/DocHeader/DocHeader.tsx:51]: You do not need to use a wrapper (div) just to add margin|padding. Pass the props down to component to pick it up.

```ts
 <DocHeaderVersions mt={1} versions={versions} />
 ```

Comment (alex-kovoy) [inline components/DocHeader/DocHeader.tsx:51]: `DocHeaderProps` -> `Props`  No need for this extra prefixes `DocHeaderXXX`, they add no value.

**Review by alex-kovoy — APPROVED** [review]
Looks good (with minor comments)

File: components/DocHeader/DocHeader.tsx (hunk 2 above)
Comment (iAdramelk) [reply components/DocHeader/DocHeader.tsx:51]: Have to argue with it. In the editor and error messages in you see a lot of Props that are extending or combining other  Props it's pretty hard to debug them. But if you see the name, it[s pretty easy to find the place of the problem.

**Pull request merged**